ROI White
ROI Black
bool is_set
---
ROI white
ROI black
